Car_rental_app/website/views.py

```diff
@@ -302,3 +307,31 @@ def checkout_requests():
 
 
 
+
+
+
+@views.route('/admin/sales', methods=['GET', 'POST'])
+def admin_sales():
```

Comment:
you can try to make a try-catch error to handle cases where the data is missing or invalid for example the code could be:
    if request.method == 'POST':
        try:
            month = int(request.form['month'])
            year = int(request.form['year'])
        except (KeyError, ValueError) as e:
            # handle invalid or missing data
            flash("Invalid input. Please select a valid month and year.", "error")
            return redirect(url_for('views.admin_sales'))